<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1098</width>
    <height>799</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1098</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionClear"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionErase"/>
   </widget>
   <widget class="QMenu" name="menuBrushSize">
    <property name="title">
     <string>BrushSize</string>
    </property>
    <addaction name="action3px"/>
    <addaction name="action5px"/>
    <addaction name="action7px"/>
    <addaction name="action9px"/>
    <addaction name="action11px"/>
    <addaction name="action13px"/>
   </widget>
   <widget class="QMenu" name="menuBrushColor">
    <property name="title">
     <string>BrushColor</string>
    </property>
    <addaction name="actionBlack"/>
    <addaction name="actionBlue"/>
    <addaction name="actionRed"/>
    <addaction name="actionYellow"/>
    <addaction name="actionGreen"/>
   </widget>
   <widget class="QMenu" name="menuBrushes">
    <property name="title">
     <string>Brushes</string>
    </property>
    <addaction name="actionSolid"/>
    <addaction name="actionAir"/>
    <addaction name="actionCalligraphy"/>
    <addaction name="actionOil"/>
    <addaction name="actionCrayon"/>
   </widget>
   <widget class="QMenu" name="menuShapes">
    <property name="title">
     <string>Shapes</string>
    </property>
    <addaction name="actionRectangle"/>
    <addaction name="actionCircle"/>
    <addaction name="actionline"/>
    <addaction name="actionArrow"/>
    <addaction name="actionRoundedRectangle"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBrushSize"/>
   <addaction name="menuBrushColor"/>
   <addaction name="menuBrushes"/>
   <addaction name="menuShapes"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionClear"/>
   <addaction name="actionExit"/>
   <addaction name="actionUndo"/>
   <addaction name="actionErase"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionRectangle"/>
   <addaction name="actionCircle"/>
   <addaction name="actionline"/>
   <addaction name="actionArrow"/>
   <addaction name="actionRoundedRectangle"/>
  </widget>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/clear.jpg</normaloff>:/images/clear.jpg</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/exit 2.png</normaloff>:/images/exit 2.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionErase">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/eraser.jpg</normaloff>:/images/eraser.jpg</iconset>
   </property>
   <property name="text">
    <string>Erase</string>
   </property>
   <property name="shortcut">
    <string>Backspace</string>
   </property>
  </action>
  <action name="action3px">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/three.png</normaloff>:/images/three.png</iconset>
   </property>
   <property name="text">
    <string>3px</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+3</string>
   </property>
  </action>
  <action name="action5px">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/five.png</normaloff>:/images/five.png</iconset>
   </property>
   <property name="text">
    <string>5px</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+5</string>
   </property>
  </action>
  <action name="action7px">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/seven.png</normaloff>:/images/seven.png</iconset>
   </property>
   <property name="text">
    <string>7px</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+7</string>
   </property>
  </action>
  <action name="action9px">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/nine.png</normaloff>:/images/nine.png</iconset>
   </property>
   <property name="text">
    <string>9px</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+9</string>
   </property>
  </action>
  <action name="action11px">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/eleven.png</normaloff>:/images/eleven.png</iconset>
   </property>
   <property name="text">
    <string>11px</string>
   </property>
  </action>
  <action name="action13px">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/thirteen.png</normaloff>:/images/thirteen.png</iconset>
   </property>
   <property name="text">
    <string>13px</string>
   </property>
  </action>
  <action name="actionBlack">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/black.png</normaloff>:/images/black.png</iconset>
   </property>
   <property name="text">
    <string>Black</string>
   </property>
  </action>
  <action name="actionBlue">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/blue.png</normaloff>:/images/blue.png</iconset>
   </property>
   <property name="text">
    <string>Blue</string>
   </property>
  </action>
  <action name="actionRed">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/red.png</normaloff>:/images/red.png</iconset>
   </property>
   <property name="text">
    <string>Red</string>
   </property>
  </action>
  <action name="actionYellow">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/yellow.png</normaloff>:/images/yellow.png</iconset>
   </property>
   <property name="text">
    <string>Yellow</string>
   </property>
  </action>
  <action name="actionGreen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/green.png</normaloff>:/images/green.png</iconset>
   </property>
   <property name="text">
    <string>Green</string>
   </property>
  </action>
  <action name="actionSolid">
   <property name="text">
    <string>Solid Brush</string>
   </property>
  </action>
  <action name="actionAir">
   <property name="text">
    <string>Air Brush</string>
   </property>
  </action>
  <action name="actionCalligraphy">
   <property name="text">
    <string>Calligraphy Brush</string>
   </property>
  </action>
  <action name="actionOil">
   <property name="text">
    <string>Oil Brush</string>
   </property>
  </action>
  <action name="actionCrayon">
   <property name="text">
    <string>Crayon Brush</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/rectangle.png</normaloff>:/images/rectangle.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionCircle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/circle.png</normaloff>:/images/circle.png</iconset>
   </property>
   <property name="text">
    <string>Circle</string>
   </property>
  </action>
  <action name="actionline">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/line.png</normaloff>:/images/line.png</iconset>
   </property>
   <property name="text">
    <string>line</string>
   </property>
  </action>
  <action name="actionArrow">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/arrow.png</normaloff>:/images/arrow.png</iconset>
   </property>
   <property name="text">
    <string>Arrow</string>
   </property>
  </action>
  <action name="actionRoundedRectangle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/roundedrectangle.png</normaloff>:/images/roundedrectangle.png</iconset>
   </property>
   <property name="text">
    <string>RoundedRectangle</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
